.github/workflows/package-apisix-deb-focal1.yml

```diff
@@ -0,0 +1,52 @@
+name: package apisix deb for ubuntu 20.04(Focal Fossa)
```

Comment:
There are two workflows for ubuntu 20.04?

Reply:
> There are two workflows for ubuntu 20.04?

yep, one is explicitly for ubuntu 20.04, the other is for testing the default image of `deb`.

Reply:
I would propose turning on more workflows for more OS distributions in other PRs.